composer.json

```diff
@@ -110,8 +110,10 @@
     "drupal/openid_connect_daccount": "1.x-dev@dev",
     "drupal/password_policy": "^4",
     "drupal/queue_ui": "^3",
+    "drupal/real_aes": "^2.5", 
```

Comment:
@sekinet813 Can we change the version to `^2` for the dev branch?  We can put the specific version to the official release.